v1.1/configure-replication-zones.md

```diff
@@ -135,16 +135,10 @@ The `zone` command and subcommands support the following [general-use](#general)
 
 Flag | Description
 -----|------------
-`--certs-dir` | The path to the [certificate directory](create-security-certificates.html). The directory must contain valid certificates if running in secure mode.<br><br>**Env Variable:** `COCKROACH_CERTS_DIR`<br>**Default:** `${HOME}/.cockroach-certs/`
-`--database`<br>`-d` | Not currently implemented.
 `--disable-replication` | Disable replication in the zone by setting the zone's replica count to 1. This is equivalent to setting `num_replicas: 1`.
 `--echo-sql` | <span class="version-tag">New in v1.1:</span> Reveal the SQL statements sent implicitly by the command-line utility. For a demonstration, see the [example](#reveal-the-sql-statements-sent-implicitly-by-the-command-line-utility) below.
 `--file`<br>`-f` | The path to the [YAML file](#replication-zone-format) defining the zone configuration. To pass the zone configuration via the standard input, set this flag to `-`.<br><br>This flag is relevant only for the `set` subcommand.
-`--host` | The server host to connect to. This can be the address of any node in the cluster. <br><br>**Env Variable:** `COCKROACH_HOST`<br>**Default:** `localhost`
-`--insecure` | Run in insecure mode. If this flag is not set, the `--certs-dir` flag must point to valid certificates.<br><br>**Env Variable:** `COCKROACH_INSECURE`<br>**Default:** `false`
-`--port`<br>`-p` | The server port to connect to. <br><br>**Env Variable:** `COCKROACH_PORT`<br>**Default:** `26257`
-`--url` | The connection URL. If you use this flag, do not set any other connection flags.<br><br>For insecure connections, the URL format is: <br>`--url=postgresql://<user>@<host>:<port>/<database>?sslmode=disable`<br><br>For secure connections, the URL format is:<br>`--url=postgresql://<user>@<host>:<port>/<database>`<br>with the following parameters in the query string:<br>`sslcert=<path-to-client-crt>`<br>`sslkey=<path-to-client-key>`<br>`sslmode=verify-full`<br>`sslrootcert=<path-to-ca-crt>`<br><br>**Env Variable:** `COCKROACH_URL`
-`--user`<br>`-u` | The user connecting to the database. Currently, only the `root` user can configure replication zones. <br><br>**Env Variable:** `COCKROACH_USER`<br>**Default:** `root`
+`--url`, `--host`, `--port`, `--user`, `--database`, `--certs-dir`, `--insecure`,  | See [Client Connection Parameters](connection-parameters.html) for details. Currently, only the `root` user can configure replication zones and the `--database` flag is not effective.
```

Comment:
I have some concerns about this approach:

1. On the one hand, I don't like bouncing users to a separate page to learn about the flags for a specific `cockroach` command. That's a burden that a reference doc should try to avoid. As an alternative, I'd prefer to leave the flags on the command pages, grouped under a new `Client Connection` header, and have a note at the top of that section calling out the option to use a url instead with a link to the new page. I'd like to hear your response to this suggestion before reviewing much more. Would also be interested in @bdarnell or @mberhault's opinion here.

2. But if we have to do this, I'd suggest we change the presentation to either:
    1. Use `<br>` to put each flag on its own line in the cell.
    2. Add a distinct section called **Client Connection** and add a sentence with a link to the page you created.